### DISCUSSION TITLE
Foldable sidebar (icons only)

### DISCUSSION THREAD

**Discussion by tooomm** [paperless-ngx/paperless-ngx#1221] [Feature Requests] [answered]
Made a quick mock-up of the proposal:

- current main view (documents)
![documents-wchrome](https://user-images.githubusercontent.com/9874850/178344274-cd89347a-db49-45cd-9335-ca6481095032.png)
- minimized down to icons for an expanded workspace
![documents-wchrome2](https://user-images.githubusercontent.com/9874850/178344282-b8e43723-6e71-41e0-b556-c05a3d1a6634.png)

<br>

I enjoy a lot that many apps have this now and allow to maximize the content part and allow for the menus to eat up less space on demand. The app stays navigable and the icons still give the key information away.

What I did not cover:
How to activate this or toggle between the two views and where it would be placed at best. Some apps use arrows like ![Untitled](https://user-images.githubusercontent.com/9874850/178345796-bc526ba6-f61d-41a0-a476-713fd958cfdf.png), or "◄" as seen in the screenshot at the bottom. Others have the known hamburger menu.

Another important part would be to show mouse over hints on the icons with the name of the option or e.g. the document or filter name.
While this could be a simple version, Adobe Reader uses a pretty cool variant in recent versions:
![Untitled](https://user-images.githubusercontent.com/9874850/178346323-39b72b10-9aef-48a1-a6a4-346e4341c45f.png)

**Comment by shamoon** [comment]
Good idea yea. I don’t have the time for it right now but maybe can circle back to it

**Comment by sukisoft** [comment]
I came up with the exact same idea this morning. Would love to see this.

**Comment by shamoon** [comment, marked as answer]
Edit: proposed in https://github.com/paperless-ngx/paperless-ngx/pull/1641

**Reply by sukisoft** [reply]
Pretty good, maybe something for a 2.0 release which may or may not include other ui tweaks. Anyways great job. Looking forward to this!
